zeekpkg/template.py

```diff
@@ -483,7 +483,7 @@ def contentdir(self):
         Returns:
             str: relative path to the content directory
         """
-        return None
```

Comment:
These blank returns at the end of the methods look off, I'd prefer if they were just gone (though I know the changes were automated)

Reply:
Good catch, thanks. I have replaced these with `pass`. I already made some manual edits here by adding empty lines between consecutive `if` blocks.